src/python/director/transformUtils.py

```diff
@@ -149,7 +145,7 @@ def frameInterpolate(trans_a, trans_b, weight_b):
     [pos_a, quat_a] = poseFromTransform(trans_a)
     [pos_b, quat_b] = poseFromTransform(trans_b)
     pos_c = pos_a *(1-weight_b) + pos_b * weight_b;
-    quat_c = botpy.quat_interpolate(quat_a,quat_b, weight_b)
+    quat_c = transformations.quaternion_slerp(quat_a, quat_b, weight_b)
```

Comment:
This replaces botpy.quat_interpolate with transformations.quaternion_slerp.  The associated test confirms that both functions give the same result (using randomly selected quaternions).